tests/bsim/bluetooth/host/misc/sample_test/test_scripts/run.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Please add a comment telling developers that if they copy this to build their own test they should remove all the explanatory comments

Reply:
They should be grownups imo and realize this is a sample.
IF they're the sort to blindly copy and paste, they will also copy the comment saying not to copy it.
Ever had some dvd rips with the FBI warning in them? :p

Reply:
Ok, just be sure to keep an eye during reviews, because given the amount of comments copy pasted around I expect this will happen :)